pruner/window.go

```diff
@@ -0,0 +1,7 @@
+package pruner
+
+import (
+	"time"
+)
+
+const Window = 30 * 86400 * time.Second // 30 days worth of seconds
```

Comment:
Should be local. It will be available through exported interface anyway

Reply:
My only concern is about where it should live as it would be used by the `full` and `light` implementations - so it needs to live somewhere as a default / param

Reply:
If interface is splitted, it could become part of `archival` / `pruned` implementation only, while not affecting `full` / `light` store prunning implementations